backend/data/src/main/kotlin/io/tolgee/service/apps/AppManifestFetcher.kt

```
@@ -0,0 +1,248 @@
+    if (icon.isNullOrEmpty()) return null
+    if (icon.length > MAX_ICON_LENGTH) {
+      throw BadRequestException(
+        Message.APP_MANIFEST_INVALID,
+        listOf("icon exceeds $MAX_ICON_LENGTH characters"),
+      )
+    }
+    if (!icon.contains('/')) return icon
+    val resolved =
+      try {
+        URI(manifest.baseUrl).resolve(icon)
+      } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)